modules/react/action-bar/lib/ActionBarList.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+
+import {commonColors, colors, space} from '@workday/canvas-kit-react/tokens';
+import {
+  composeHooks,
+  createSubcomponent,
+  ExtractProps,
+  styled,
+  StyledType,
+} from '@workday/canvas-kit-react/common';
+import {HStack} from '@workday/canvas-kit-react/layout';
+import {
+  useOverflowListMeasure,
+  useListRenderItems,
+  useListResetCursorOnBlur,
+} from '@workday/canvas-kit-react/collection';
+
+import {useActionBarModel} from './useActionBarModel';
+import {ActionBar} from './ActionBar';
+
+// Use `Partial` here to make `spacing` optional
+export interface ActionBarListProps<T = unknown>
+  extends Partial<ExtractProps<typeof HStack, never>> {
+  /**
+   * If items are passed to a `ActionBarModel`, the child of `ActionBar.List` should be a render prop. The
+   * List will determine how and when the item will be rendered.
+   *
+   * @example
+   * <ActionBar.List>
+   *   {(item) => <ActionBar.Item key={item.id} name={item.name}>{item.text}</ActionBar.Item>}
+   * </ActionBar.List>
+   */
+  children: ((item: T) => React.ReactNode) | React.ReactNode;
+  overflowButton?: boolean;
```

Comment:
curious what others think, should we rename this something like `shouldOverflow` or `showOverflowButton`